dts/bindings/espi/gpio-espi.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2018, Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO eSPI parent node
+
+compatible: "gpio-espi"
```

Comment:
This seems a little unusual.  I think we'd normally do something like:
```
properties:
  pwrgrd:
    type: phandle-array
    # ...
  resume:
    type: phandle-array
    #...
```

Especially since the driver hard codes the property names to have `_PWRGD_` and `_RESUME_` in them, and there's no documentation here specifying that expectation.

Is there a reason this approach is necessary?  cc @galak

Reply:
Used slightly different approach in dts and added description, let me know if the pin definitions should be leave into Kconfig.